### PR TITLE
Sg/minor fixes 3

### PR DESCRIPTION
**Pull request by Kink3d**
### Purpose of this PR
Fixes the following bugs:

Rotate About Axis node enum is backwards
https://fogbugz.unity3d.com/f/cases/1115422/

Convert to Property / Convert to Inline breaks Group Inclusion
https://fogbugz.unity3d.com/f/cases/1115108/

Flip node needs better UX/feedback
https://fogbugz.unity3d.com/f/cases/1108468/

BooleanShaderProperty uses wrong property attribute (creates keywords)
https://fogbugz.unity3d.com/f/cases/1109525/

Normal Unpack Object Space not working
https://fogbugz.unity3d.com/f/cases/1108780/

Constant/Default port values on several nodes arent used in the generated shader code
https://fogbugz.unity3d.com/f/cases/1108777/

Channel Mask Duplication Bug
https://fogbugz.unity3d.com/f/cases/1108772/

Alpha control value on Flip is not being serialized
https://fogbugz.unity3d.com/f/cases/1108480/

IsInfinite & IsNan uses dynamic vector; misleading
https://fogbugz.unity3d.com/f/cases/1108410/

Can right-click nodes in a Sub Graph and convert to Sub Graph
https://fogbugz.unity3d.com/f/cases/1097229/

Transform Matrix: Inverse Project & Inverse View Project are Broken
https://fogbugz.unity3d.com/f/cases/1115116/

Term "Shader Graph" is not capitalized correctly when shown on save prompt
https://fogbugz.unity3d.com/f/cases/1101069/

---
### Release Notes
Check changelog

---
### Testing status
**Katana Tests**: Running...

**Manual Tests**: Checked reproductions in FB issues. QA will verify.

**Automated Tests**: N/A

Any test projects to go with this to help reviewers? N/A

---
### Overall Product Risks
**Technical Risk**: Low

**Halo Effect**: Low

---
### Comments to reviewers
N/A

### CODE REVIEW

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Reverted the visual styling of various graph elements to their previous correct states.
 - Previews now repaint correctly when Unity does not have focus.
 - Code generation now works correctly for exposed Vector1 shader properties where the decimal separator is not a dot.
+- The `Rotate About Axis` node's Modes now use the correct function versions.
+- Converting nodes between property and inline now preserves grouping.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Unity now preserves grouping when you convert nodes between property and inline.

Comment (Kink3d) [reply com.unity.shadergraph/CHANGELOG.md]: "Unity" or "Shader Graph"? Otherwise makes sense.

File: com.unity.shadergraph/CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- The `Flip` node now greys out labels for inactive controls.
+- The `Boolean` property type now uses the `ToggleUI` property attribute as to not generate keywords.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Does this mean:
The `Boolean` property type now uses the `ToggleUI` property attribute, so as to not generate keywords.
or
The `Boolean` property type now uses the `ToggleUI` property attribute in a way that does not generate keywords.
?

File: com.unity.shadergraph/CHANGELOG.md (hunk 1 above)
Comment (sharlenet) [reply com.unity.shadergraph/CHANGELOG.md]: Better to be specific, so:
Shader Graph now preserves grouping when you convert nodes between property and inline.

File: com.unity.shadergraph/CHANGELOG.md [hunk 3]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- The `Normal Unpack` node no longer generates errors in Object space.
+- The `Split` node now uses values from its default Port input fields.
+- The `Channel Mask` node now allows multiple node instances without error.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Minor change:
The `Channel Mask` node now allows multiple node instances, and no longer generates any errors.

File: com.unity.shadergraph/CHANGELOG.md [hunk 4]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- The value for the Alpha control on the `Flip` node is now serialized.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Serialized the value for the Alpha control on the `Flip` node.
or
Serialized the Alpha control value on the `Flip` node.

File: com.unity.shadergraph/CHANGELOG.md [hunk 5]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- The `Is Infinite` and `Is NaN` nodes now use `Vector 1` input ports, however the output remains the same.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: The `Is Infinite` and `Is NaN` nodes now use `Vector 1` input ports, but the output remains the same.

File: com.unity.shadergraph/CHANGELOG.md [hunk 6]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- You can no longer attempt to convert nodes inside a `Sub Graph` in to a `Sub Graph`, which previously caused errors.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Consider if "attempt to" is really necessary. "in to" should be one word. Also, wasn't sure if you converted multiple nodes into a `Sub Graph`, or just one node into a `Sub Graph`.
Suggestion:
You can no longer convert a node inside a `Sub Graph` into a `Sub Graph`, which previously caused errors.

File: com.unity.shadergraph/CHANGELOG.md [hunk 7]
```diff
@@ -26,6 +26,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- The `Transformation Matrix` node's Inverse PRojection and Inverse View Projection modes no longer produce errors.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Changed "R" in Projection to lower case:
The `Transformation Matrix` node's Inverse Projection and Inverse View Projection modes no longer produce errors.

**Pull request merged**